### PR TITLE
OPL updates

### PR DESCRIPTION
**Pull request by rfomin**
Unfinished work on OPL3 integration.

Is this PR the way I should work on it?

### CODE REVIEW

**Comment by kraflab** [comment]
Yes this is the way.

**Comment by rfomin** [comment]
All updates from Chocolate Doom are in, OPL3 mode is enabled by default.

Maybe we should rename the OPL2 player menu item to just OPL.

File: prboom2/src/dsda/configuration.c [hunk 1]
```diff
@@ -756,6 +756,10 @@ dsda_config_t dsda_config[dsda_config_count] = {
     "mus_opl_gain", dsda_config_mus_opl_gain,
     dsda_config_int, 0, 1000, { 50 }
   },
+  [dsda_config_mus_opl_opl3mode] = {
+    "mus_opl_opl3mode", dsda_config_mus_opl_opl3mode,
```
Comment (kraflab) [inline prboom2/src/dsda/configuration.c:760]: Why would someone want to disable this? What is the difference exactly?

**Comment by kraflab** [comment]
> Maybe we should rename the OPL2 player menu item to just OPL.

Yes I think that makes sense.

File: prboom2/src/dsda/configuration.c (hunk 1 above)
Comment (rfomin) [reply prboom2/src/dsda/configuration.c:760]: OPL2 sounds worse and only mono. But most of the popular sound cards in the early 90s were OPL2.

Comment (kraflab) [reply prboom2/src/dsda/configuration.c:760]: I see, so it's there for people who want the crustiest option :^)

**Comment by kraflab** [comment]
Anything else needed or is it good to go?

**Comment by rfomin** [comment]
> Anything else needed or is it good to go?

I think it's done.

**Comment by kraflab** [comment]
Thanks!

**Pull request merged**